python_modules/dagster/dagster/_core/definitions/decorators/asset_decorator.py

```diff
@@ -343,6 +364,16 @@ def __call__(self, fn: Callable) -> AssetsDefinition:
 
             op_required_resource_keys = decorator_resource_keys - arg_resource_keys
 
+            code_origin_tag = (
```

Comment:
It's a little weird to have a variable that's a dictionary have a name that's a singular noun. Also, what are the tradeoffs of having a single tag with a JSON blob under it vs. separate tags for the file and line?

Reply:
actually we don't even use this asset metadata in the UI, I'll just remove it